### ISSUE TITLE
Add health checks + kubernetes liveness/readiness probes

### ISSUE THREAD

**Issue by 0dragosh** [RemiRigal/Plex-Auto-Languages#19]
It would be great to have at least one HTTP endpoint that we can use for a health check at deployment.

Could be as simple as:
```python
@app.route('/healthz')
def live():
  return "Live!"

@app.route('/readyz')
def ready():
  return "Ready!"
  ```
after config is loaded. (example is using flask)

**Comment by RemiRigal** [comment]
Thank you for opening the issue, I just merged #21 that adds the health-check HTTP server.

You can use one of those endpoints:
- `/` or `/health` that returns a response code 200 if healthy, 400 otherwise. The response body is: `{"healthy": true/false}`
- `/ready` that returns a response code 200 if ready, 400 otherwise. The response body is: `{"ready": true/false}`

**Comment by RemiRigal** [comment]
FYI I've changed the port on which the health-check server is running to `9880` in order to avoid the need for elevated privileges when running the application without Docker. This change will be included in the next release.
